Content.Shared/Forensics/Events.cs

```diff
@@ -24,3 +24,30 @@ public ForensicPadDoAfterEvent(string sample)
 
     public override DoAfterEvent Clone() => this;
 }
+
+[Serializable, NetSerializable]
+public sealed partial class CleanForensicsDoAfterEvent : SimpleDoAfterEvent
+{
+}
+
+/// <summary>
+/// An event to apply DNA evidence from a donor onto some recipient.
+/// </summary>
+[ByRefEvent]
+public record struct TransferDnaEvent()
+{
```

Comment:
is there a reason this isnt just a method call in `ForensicsSystem`? why the indirection thru raising an event?

Reply:
Sloth asked for an event to keep forensics separate from chemistry. I went ahead and used the event everywhere to keep it consistent. I can swap most of them to a method call if we would like. It would just be chemistry and maybe a couple places in Shared that would use the event then.

Reply:
I went ahead and updated those places that didn't need to be an event. Thank your for the review!